pkg/rtcp/transport_layer_nack.go

```diff
@@ -0,0 +1,114 @@
+package rtcp
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+)
+
+// The TransportLayerNack packet informs the encoder about the loss of a transport packet
+type TransportLayerNack struct {
+	// SSRC of sender
+	SenderSSRC uint32
+
+	// SSRC of the media source
+	MediaSSRC uint32
+
+	Nacks []struct {
+		// ID of lost packets
+		PacketID uint16
+
+		// Bitmask of following lost packets
+		BLP uint16
```

Comment:
For someone who hasn't read the spec before this name might be hard to understand. What if it was called `LostPackets` or something?

You could even make a type around the bitmap int so the end user doesn't have to do the bit-shifting themselves.

```go
  ...

  LostPackets PacketBitmap
}

type PacketBitmap uint16

func (p PacketBitmap) Lost(i int) bool {
 ...
}
```

Reply:
Love this idea, i'll implement it.